exercises/binary-search/.meta/hints.md

```diff
@@ -0,0 +1,17 @@
+## Hints
+
+[Slices](https://doc.rust-lang.org/book/second-edition/ch04-03-slices.html) have additionally to
+the normal element access via indexing (slice[index]) many useful functions like 
+[split_at](https://doc.rust-lang.org/std/primitive.slice.html#method.split_at) or [getting 
+subslices](https://doc.rust-lang.org/std/primitive.slice.html#method.get) (slice[start..end]).
+
+Can can solve this exercise by just using boring old element access via indexing, but maybe the
```

Comment:
Repeated word: `Can can solve this exercise...`

Reply:
Thanks for spotting this typo. I have fixed it now. 
In order to get one clean commit I have amended the fix to my last commit. I'm not sure if this is good or bad practice, because it makes probably a review of my fix more complicated for you?

Reply:
Disclaimer: I'm not a project maintainer, so don't take this as gospel.

What I've seen from the maintainers is that they tend to squash multi-commit PRs into a single commit anyway, so I don't think it matters either way.

Reply:
> In order to get one clean commit I have amended the fix to my last commit. I'm not sure if this is good or bad practice, because it makes probably a review of my fix more complicated for you?

Doesn't matter to me. I check out what I review locally and so I'll always be able to diff PR current contents versus what I last reviewed.

> squash multi-commit PRs into a single commit

This must only be done when the PR would make sense as a single commit. If the PR contains two logical changes, the maintainers must not squash.